pyproject.toml

```
@@ -15,6 +15,10 @@ ignore = [
     "noxfile.py",
 ]
 
+[tool.isort]
+known_first_party = "env,pybind11_cross_module_tests,pybind11_tests,"
+profile = "black"
```

```suggestion
profile = "black"
multi_line_output = 3
```

I think this is slightly more black-like, but was added after the black profile was created.

---

Ahh, you are correct in your commit message, this is was only true on older black versions - it's now part of the standard black Profile. Please feel free to drop. We only support the linter versions as specified in the pre-commit file.